src/views/Dashboard/components/Widgets/SendWidget/SendWidget.tsx

```diff
@@ -1,22 +1,25 @@
 import React, { memo, useCallback, useState } from "react";
 import { Button, Grid, Stack, styled, Typography } from "@mui/material";
 import useAPIRouter from "hooks/useAPIRouter";
-import JUPAssetSearchBox from "components/JUPAssetSearchBox";
 import JUPInput from "components/JUPInput";
 
 const SendWidget: React.FC = () => {
-  const [toAddress, setToAddress] = useState<string>("");
+  const [toAddress, setToAddress] = useState<string>();
   const [sendQuantity, setSendQuantity] = useState<string>();
   const { sendJUP } = useAPIRouter();
 
   const handleSend = useCallback(async () => {
-    if (sendJUP === undefined || sendQuantity === undefined || toAddress === "") {
+    if (toAddress === undefined || sendJUP === undefined || sendQuantity === undefined) {
       return;
     }
 
-    const result = await sendJUP(toAddress, sendQuantity);
-
-    console.log("sendWidget sendJUP result:", result);
+    try {
+      const result = await sendJUP(toAddress, sendQuantity);
```

Comment:
`sendJUP` should probably sanitize the input of amount soon. essentially a function that ensures that the value sent is within the allowable ranges that JUP handles. i'm assuming only positive values are allowed, up to the maximum, with a max number of decimals. 

The UI can help ensure the user does a better job of entering this, but ultimately the send function needs to verify since a user could change the html in their browser or devs could mess up and not implement the same UI in every place sendJUP is used.

the layer that attempts to catch this right now is the api and i have to assume that it does a good job of this or problems would have been found by now. so this isn't the most critical issue but it should be simple to solve in the near future.

Reply:
agreed, i made a required4delivery issue: https://github.com/Up-North-Coding/jovian/issues/80